spaces/importers/tprek_unit_importer.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from dataclasses import asdict
+
+from django.db.models import QuerySet
+from django.db.transaction import atomic
+
+from opening_hours.models import OriginHaukiResource
+from opening_hours.utils.hauki_api_client import HaukiAPIClient
+from opening_hours.utils.hauki_api_types import HaukiAPIResource
+from spaces.importers.tprek_api_client import TprekAPIClient, TprekLocationData, TprekUnitData
+from spaces.models import Location, Unit
+from utils.sentry import SentryLogger
+
+logger = logging.getLogger(__name__)
+
+
+class TprekUnitImporter:
+    def __init__(self) -> None:
+        self.updated_units_count = 0
+        self.units_for_hauki_import = []
+
+    @atomic
```

Comment:
This atomic is also a little too broad. Should it be scoped to the `_update_unit` method?